README.md

```diff
@@ -90,7 +90,7 @@ image we provide:
 docker run
   -e MARGE_AUTH_TOKEN="$(cat marge-bot.token)" \
   -e MARGE_SSH_KEY="$(cat marge-bot-ssh-key)" \
-  smarketshq/marge-bot:0.1.0 \
+  smarketshq/marge-bot \
```

Comment:
Yup, agreed. Just saying that a) we should add something to the Makefile or whatever that does it automatically, and b) until we do, you will need to tag manually with latest (as well) before pushing.